deploy/clowdapp.yml

```diff
@@ -0,0 +1,194 @@
+---
```

Comment:
We have a cronjob that runs as part of the gateway.  Should the cronjob be listed in this yml file?

I do not see where/how the switch (it runs the job-receiver process) gets started?

Reply:
The switch podSpec starts on line 69, and i added the cronjob